src/main/java/seedu/duke/company/CompanyList.java

```diff
@@ -51,4 +51,8 @@ public void loadSampleCompanyInformation() throws InputMismatchException {
         companyList.add(sampleCompany2);
         companyList.add(sampleCompany3);
     }
+
+    public void purgeData(){
```

Comment:
perhaps a space can be added between `)` and `{` for the codestyle to be consistent throughout